Installation/cmake/modules/FindSuiteSparse.cmake

```diff
@@ -0,0 +1,437 @@
+## CMake file to locate SuiteSparse and its useful composite projects
```

Comment:
I'm not sure what you mean; SuiteSparse allows to use another solver and gives a decent speed-up in the resolution of Orbifold systems. It's annoying to find and link so I added a FindSuiteSparse.cmake to help that.